lib/mohawk/navigation.rb

```diff
@@ -10,7 +10,7 @@ def on(cls, extra = {}, &block)
       block.call screen if block
       screen
     rescue Mohawk::Waiter::WaitTimeout => e
-      raise e.class, "Unable to locate '#{cls}'", e.backtrace
+      raise e.class, e.message.sub(/window/, cls.to_s), e.backtrace
     end

```

Comment:
I moved the primary message to https://github.com/leviwilson/mohawk/pull/13/commits/c97fd6ce82e5c22649c9e98d9e1aa391cefad556#diff-1d3cb00bba3f9c3b064753fd84559b7eR55 and updated it to use the context parameter. I really wanted to get the class name into the error message which is why I added the sub here. If the footprint here is not worth the benefit then we could remove the changes to this file.